header.php

```diff
@@ -23,7 +23,7 @@
 // to a valid X-Frame-Options header value or false
 if (defined('FORUM_FRAME_OPTIONS'))
 {
-	if (preg_match('/^(?:allow-from|deny|sameorigin)/i', FORUM_FRAME_OPTIONS) === 1)
+	if (preg_match('%^(?:allow-from|deny|sameorigin)%i', FORUM_FRAME_OPTIONS))
```

Comment:
@franzliedke Yeah, me too. I just escape any literal slashes I want in my regex to avoid problems.